src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const couple = (sex === 'm')
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
```

Comment:
better to move text to the new line, it will look much more readable
```suggestion
      <h2 className="Person__name">
        {`My name is ${name}`}
      </h2>
```